usaspending_api/etl/es_award_template.json

```diff
@@ -667,6 +667,12 @@
             "type": "keyword"
           }
         }
+      },
+      "program_activity_names": {
+        "type": "keyword"
```

Comment:
Any particular reason to make this exactly "keyword" and not the "text" and "keyword" pattern of some other keys?

Reply:
I haven't heard of any plans to use these fields for partial searching so I didn't include the `text` version. I'll add it though to be consistent with the other fields.